capture/app.py

```diff
@@ -46,3 +68,24 @@ def main():
         logger.debug(e)
         parser.print_help()
         # raise
+
+    try:
+        logging.basicConfig(level=logging.INFO)
+        logger = logging.getLogger(__name__)
+        extension = [".fastq", ".fq", ".fastq.gz", ".fq.gz", ".sam", ".bam"]
+        if not args.input_file.split(",").endswith(extension):
+            logger.info("please use a correct input file")
+            sys.exit(0)
+        else:
+            __split__(args)
+    except AttributeError as e:
+        logger.debug(e)
+        parser.print_help()
+        # raise
+
+    try:
+        if not os.path.exists("tempdir"):
```

Comment:
the temporary directory should **absolutely** not be at the base directory where the script is run. It should be int the output directory of the software. I'd add an `--output` argument and put every file `capture` creates in it.